### DISCUSSION TITLE
Cancel Return Path

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3203] [Development discussions]
I want to cancel the return path part below. How can I do this? In other words, I want to write uyrsozlugu.com in the signer and sender section.
![image](https://github.com/user-attachments/assets/4ba837ed-46db-4240-bdbe-8f3f3691c454)

**Comment by willpower232** [comment]
Have a read of https://github.com/orgs/postalserver/discussions/3195

**Comment by JohnHammell** [comment]
I started the discussion that willpower232 linked to. Still trying to iron some things out in general with Postal for how I'm trying to use it with a new project.

As far as I can tell, "psrp" is set via the custom_return_path_prefix in postal.yml, but it doesn't appear to be activated unless you actually have it configured in your DNS. I noticed that without the DNS setting "psrp" custom return path is not used even if it is setup in the custom_return_path_prefix variable.

To use just your domain, example.com, as the return path, I believe you can just change the default setting of "return_path_domain: rp.postal.example.com" to be "return_path_domain: example.com" and that should do it.

**Reply by TR-SametOzen** [reply]
Hello @JohnHammell ,

Can I reach you via your private e-mail?

**Reply by JohnHammell** [reply]
Sure, if you want to post your email address I'll send you a message so we can chat. Would be good if github had a private messaging feature.

**Reply by JohnHammell** [reply]
Just wanted to let you know I got your address from your post & sent an email, in case you wanted to edit/delete your email out of the above message.

**Comment by plancked** [comment]
Hi,

I'm facing the same problem here.

Xenforo forum app creates return paths in the form of:

bounce+Bbfa37e1d+emailaddress=icloud.com@mydomain.com

But then Postal goes ahead and replaces that with:

Return-path: <qbgvvu@psrp.mydomain.com>

Which is a problem, because the Automatic bounce handling does not work for xenforo.

Did you end up solving this issue?

Thank you.

**Reply by TR-SametOzen** [reply]
Hello,

Unfortunately, the return path continues to appear in the sender name, but I couldn't figure it out.

**Comment by plancked** [comment]
What about not using the return path CNAME?
Will that hurt deliverability?
